src/main/java/org/eclipse/cargotracker/interfaces/FacesConfiguration.java

```diff
@@ -1,10 +1,9 @@
 package org.eclipse.cargotracker.interfaces;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.faces.annotation.FacesConfig;
-import javax.faces.annotation.FacesConfig.Version;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.faces.annotation.FacesConfig;
 
 /** Jakarta Faces configuration. * */
-@FacesConfig(version = Version.JSF_2_3)
+@FacesConfig()
```

Comment:
Isn't the version specification still needed? If we omitted it previously, the flow scope stopped working I believe?

Reply:
The version is marked as `@Deprecated` in Faces 4.0

Reply:
OK. As long as everything still works, we can leave this be. Maybe you can see if this file is even needed any more?

Reply:
Did you check if this file can simply be deleted now or is it still needed somehow?

Reply:
<p class="changed_added_2_3">
 * The presence of this annotation on <span class="changed_modified_4_0">a class</span> deployed within an application 
 * <span class="changed_modified_4_0">guarantees activation of Jakarta Faces and its CDI specific features, even when 
 * <code>/WEB-INF/faces-config.xml</code> is absent and <code>FacesServlet</code> is not explicitly registered</span>.
 * </p>